plugin/pkg/scheduler/factory/factory.go

```diff
@@ -312,7 +312,58 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 		}
 	}
 
-	pluginArgs := PluginFactoryArgs{
+	predicateFuncs, err := f.GetPredicates(predicateKeys)
```

Comment:
nit: can you please create pluginArgs first, and simply pass them here and to getPriorityFunctionConfigs?
Currently you are duplicating some code, unnecessary.